### PR TITLE
La quiche Lorraine

### PR DESCRIPTION
**Pull request by padenot**

### CODE REVIEW

File: quiche-lorraine.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+_vinaigrette_ with finely chopped dill works well with this dish.
+
+Quiches keep wonderfully in the fridge for a few days, and are excellent cold
+the next day, but can also be re-heated.
+
+## Bonus
+
+Anne-Sophie Pic (8 Michelin stars) says that adding a generous table spoon of
+[Savora](https://fr.wikipedia.org/wiki/Savora) (a kind of mustard that is not
+spicy), about 20g, is a nice twist, and she is of course right.
```
Comment (Cgg) [inline quiche-lorraine.md]:
```suggestion
Anne-Sophie Pic (8 Michelin stars) says that a nice twist is to add a generous table spoon &ndash; about 20g &ndash; of
[Savora](https://fr.wikipedia.org/wiki/Savora) to the _migaine_. Of course, she is right.
```

File: quiche-lorraine.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+---
+layout: default
+title: quiche lorraine
+---
+
+# La quiche Lorraine
+
+This is a recipe for a *real* _quiche Lorraine_. It doesn't have cheese. Keep
```
Comment (Cgg) [inline quiche-lorraine.md:8]:
```suggestion
This is a recipe for a _real quiche Lorraine_. It doesn't have cheese. Keep
```

File: quiche-lorraine.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Unroll the pastry in a pie dish (possibly still on its paper, butter the dish
+generously if not using paper, and don't forget to prick the pastry if using
+puff pastry). Push with your fingers all around the crust area so that it's
+sticking to the side of the dish and stays upright. Don't skip this step, the
+moisture of the _migaine_ has a tendency to make the pastry fold back inward,
+while in the oven (which is not really fixable), and this makes for a messy
+crust, which is, everybody agrees, the best part.
+
+Pour the _migaine_ in the dish. At this stage, you might want to re-distribute
```
Comment (Cgg) [inline quiche-lorraine.md:58]: Pas de cuisson à blanc?

File: quiche-lorraine.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+can of course use a kitchen mixer or the like with a whisking attachment. Near
+the end of the whisking process, add the _lardons_ and whisk again.
+
+Unroll the pastry in a pie dish (possibly still on its paper, butter the dish
+generously if not using paper, and don't forget to prick the pastry if using
+puff pastry). Push with your fingers all around the crust area so that it's
+sticking to the side of the dish and stays upright. Don't skip this step, the
+moisture of the _migaine_ has a tendency to make the pastry fold back inward,
+while in the oven (which is not really fixable), and this makes for a messy
+crust, which is, everybody agrees, the best part.
```
Comment (Cgg) [inline quiche-lorraine.md]:
```suggestion
sticking to the side of the dish and stays upright. Don't skip this step, as the
moisture of the _migaine_ has a tendency to make the pastry fold back inward
while in the oven. This cannot really be fixed afterwards and makes for a messy crust,
ruining what is arguably the best part of the _quiche_.
```

File: quiche-lorraine.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+incorporating air bubbles in the filling, which lowers the density of the quiche
+and is key to the success of this recipe.
+
+Lower the oven temperature to 180°C, and place the quiche low in the oven, for
+about 35 minutes. If everything goes right, the _migaine_ should raise above the
+level of the pastry near the end of the cooking time. Pay attention to the
+coloration pattern you will see at the top of the filling, and rotate the quiche
+without having the oven open for too long so that the coloration is even. This
+depends on the oven and the shape of the heating element and might not be
+necessary.
```
Comment (Cgg) [inline quiche-lorraine.md]:
```suggestion
level of the pastry near the end of the cooking time. Pay attention to the
coloration pattern visible at the top of the filling; depending on how uniformly
your oven heats, you may have to rotate the quiche to make the coloration
even. Try to keep the oven door open for as little as possible while doing so.
```

**Review by Cgg — COMMENTED** [review]
Thanks for contributing :pray: 
Left a couple of suggestions.

Ideally we should mention what to do with the egg whites (visitandine or financiers are usual suspects in the Geiger household, but meringues would work too), eg in the bonus section

File: quiche-lorraine.md (hunk 3 above)
Comment (padenot) [reply quiche-lorraine.md:58]: non, on cuit à blanc lorsqu'on cuit peu, par exemple pour une tarte aux fraises. là on cuit 35min en bas de four c'est asses.

**Pull request merged**